edge/evtstreams/cpu2evtstreams/PolicyRegister.md

```diff
@@ -130,48 +87,64 @@ hzn exchange service listpolicy <published-cpu2evtstreams-service-name>
 
-- Below is the `horizon/business_policy.json` file used for this example:
+- Below is the `business_policy.json` file you grabbed in step one:
 
 ```bash
 {
-  "label": "$SERVICE_NAME Business Policy for $ARCH",
+  "label": "ibm.cpu2evtstreams Business Policy for $ARCH",
   "description": "A Horizon Business Policy example to run cpu2evtstreams",
   "service": {
-    "name": "$SERVICE_NAME",
-    "org": "$HZN_ORG_ID",
+    "name": "ibm.cpu2evtstreams",
+    "org": "IBM",
     "arch": "$ARCH",
     "serviceVersions": [
       {
-        "version": "$SERVICE_VERSION",
+        "version": "1.4.3",
         "priority":{}
       }
     ]
   },
-  "properties": [
-  ],
+  "properties": [],
   "constraints": [
     "os == \"Mojave\"",
     "model == \"Mac\" OR model == \"Pi3B\""
   ],
-  "userInput": []
+  "userInput": [
+    {
+      "serviceOrgid": "IBM",
+      "serviceUrl": "ibm.cpu2evtstreams",
+      "serviceVersionRange": "[0.0.0,INFINITY)",
+      "inputs": [
+        {
+          "name": "EVTSTREAMS_API_KEY",
+          "value": "$EVTSTREAMS_API_KEY"
+        },
+        {
+          "name": "EVTSTREAMS_BROKER_URL",
+          "value": "$EVTSTREAMS_BROKER_URL"
+        },
+        {
+          "name": "EVTSTREAMS_CERT_ENCODED",
+          "value": "$EVTSTREAMS_CERT_ENCODED"
+        }
+      ]
     }
   ]
 }
 ```
 - This simple example of a Business Policy doesn't provide any `properties`, but it does have two `constraints` that are satisfied by the `properties` set in the `horizon/node_policy.json` file, so this Business Policy should successfully deploy our Service onto the Edge Node.
 
-- At the bottom, the userInput section has the same purpose as the `horizon/userinput.json` files provided for other examples if the given services requires them. In this case the cpu2evtstreams service defines the configuration variables needed to send the data to IBM Event Streams. Though, for this example we have left the `userInput` section blank and will use the same `userinput.json` file we used before during pattern registration. 
-
-1. To publish this Business Policy to the Exchange and get this Service running on the Edge Node edit the `horizon/business_policy.json` file to correctly identify your specific Service name, org, version, arch, etc. When your Business Policy is ready, run the following command to publish it, giving it a memorable name (cpu2evtstreamsPolicy in this example):
+- At the bottom, the userInput section has the same purpose as the `horizon/userinput.json` files provided for other examples if the given services requires them. In this case the cpu2evtstreams service defines the configuration variables needed to send the data to IBM Event Streams. 
 
+1. Set the architecture of the Edge Node you want to deploy the `ibm.cpu2evtstreams` service to `(arm, arm64, amd64)`:
 ```bash
-hzn exchange business addpolicy -f horizon/business_policy.json cpu2evtstreamsPolicy
+export ARCH=<edge-node-arch>
```

Comment:
Instead of this can they use `$(hzn architecture)` ?

Reply:
done (so much better!)